From 0406a2216b64d0599008d5e3b5ade849ad178d70 Mon Sep 17 00:00:00 2001
From: Pascal Paillet <p.paillet@st.com>
Date: Wed, 3 May 2023 14:28:04 +0200
Subject: [PATCH 0095/1141] arm64: defconfig: Enable SCMI CPUFREQ for STM32MP25

Enable ARM_SCMI_CPUFREQ to be able to use performance
domains on STM32MP25.

Signed-off-by: Pascal Paillet <p.paillet@st.com>
Change-Id: I1e4d71a4223496fe00e5539cb8b485fb81ad9d94
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/304312
Reviewed-by: Patrick DELAUNAY <patrick.delaunay@foss.st.com>
Domain-Review: Patrick DELAUNAY <patrick.delaunay@foss.st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
---
 arch/arm64/configs/fragment-02-defconfig-addons.config | 1 +
 1 file changed, 1 insertion(+)

--- a/arch/arm64/configs/fragment-02-defconfig-addons.config
+++ b/arch/arm64/configs/fragment-02-defconfig-addons.config
@@ -1,5 +1,6 @@
 #  ALPHABETICALLY SORTED: $LC_ALL=C sort fragment-02-defconfig-addons.config -o fragment-02-defconfig-addons.config
 CONFIG_ARCH_STM32=y
+CONFIG_ARM_SCMI_CPUFREQ=y
 CONFIG_ARM_SMC_WATCHDOG=y
 CONFIG_DYNAMIC_DEBUG=y
 CONFIG_REGULATOR_ARM_SCMI=y
